django_irods/storage.py

```diff
@@ -424,7 +425,13 @@ def checksum(self, full_name):
             raise ValidationError("{} cannot be found in iRODS to retrieve "
                                   "checksum".format(obj_name))
         # remove potential '\n' from stdout
-        return stdout.rstrip('\n')
+        checksum = stdout.rstrip('\n')
```

Comment:
Now that you are at it, you can change the statement above to ```checksum = stdout.strip('\n')``` since a starting ```\n``` could be present under some special circumstances.